README.md

```diff
@@ -17,14 +15,28 @@ This is based on some of the work from the DSCO docker build, nearly all of the
 -   Images directly from docker hub, no build required
 -   Slimmed down images by using build stages and slim parent image, removes unnecessary files from images
 
-## Docker Tags
+Additionally, this fork features the following improvements:
 
-[Docker hub](https://hub.docker.com/r/coolacid/misp-docker) builds the images automatically based on git tags. I try and tag using the following details
+-   ARM (Apple M1) support
+-   Fix and improve support for cron jobs
+-   Fix Supervisor handling of entrypoints
+-   Make schema update repeatable and completely offline
+-   Fix missing MISP modules dependencies
+-   New Background Job system, see https://github.com/MISP/MISP/blob/2.4/docs/background-jobs-migration-guide.md
+-   Automatic configuration of MISP modules (see `entrypoint_internal.sh`)
+-   Automatic configuration of sync servers (see `entrypoint_internal.sh`)
+-   Automatic configuration of organizations (see `entrypoint_internal.sh`)
+-   Autoamtic configuration of authentication keys (see `entrypoint_internal.sh`)
 
-***v\[MISP Version]\[Our build version]***
+As a result, this image is not for everybody and does not (and will not) fit every use case.
+Nevertheless the underlying spirit of this fork is to allow "repeatable deployments", and all pull requests in this direction will be merged.
 
--   MISP version is the MISP tag we're building
--   Our build version is the iteration for our changes with the same MISP version
+## Versioning
+
+GitHub builds the images automatically and pushes them to [Docker hub](https://hub.docker.com/r/ostefano/misp-docker). We do not use tags and versioning works as follows:
+
+-   MISP (and modules) version specified inside the `template.env` file
+-   Docker images are tagged based on the commit hash
```

Comment:
We should specify that `core` and `modules` are tagged as `core-commit-sha1[0:7]` and `modules-commit-sha1[0:7]` respectively. The latest images would still have the additional tags `core-latest` and `modules-latest`

Reply:
ACK